packages/rollup/index.bzl

```diff
@@ -143,6 +143,7 @@ function parseStatusFile(p) {
   return results;
 }
 
+// This undefined variable will be replaced with the full path during the build.
```

Comment:
Could maybe say "the undefined 'bazel_version_file' variable" to be a bit more clear 🤷

Reply:
that's a good idea but I don't want to wait for CI again. Maybe I should make CI faster? :)

Reply:
yes you should :P